cmake/FindSourceLocation.cmake

```diff
@@ -41,6 +41,6 @@ if (NOT (TARGET SourceLocation::source_location))
        (NOT CxxSourceLocation_IMPLEMENTS_CWG2631))
     set_target_properties (SourceLocation::source_location
       PROPERTIES
-        INTERFACE_COMPILE_DEFINITIONS SEASTAR_BROKEN_SOURCE_LOCATION)
+        PUBLIC SEASTAR_BROKEN_SOURCE_LOCATION)
```

Comment:
I don't understand why we're even here, [clang-16 supports this](https://github.com/llvm/llvm-project/issues/56379#issuecomment-1379478206)

Reply:
Yup, commit has been dropped